src/containers/ProjectContentsContainer.jsx

```diff
@@ -53,6 +59,9 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <p>{this.state.translationText}</p>
             <input autoFocus placeholder="Translate some text" />
+            <DefaultButton onClick={this.handleSubmit}>
+              Submit
+            </DefaultButton>
           </ModalBody>
           <ModalFooter>
             <DefaultButton onClick={() => this.setState({modalOpen: false})}>
```

Comment:
I just ran `npm install zooniverse-readymade` as an experiment, and it updated both `package.json` and `package-lock.json`. So `npm install module-name` should be safe.